core/src/main/scala/org/apache/spark/storage/StorageStatusListener.scala

```diff
@@ -17,26 +17,55 @@
 
 package org.apache.spark.storage
 
+import java.util.concurrent.TimeUnit
+
+import scala.collection.JavaConversions.collectionAsScalaIterable
 import scala.collection.mutable
+import scala.language.reflectiveCalls
```

Comment:
I am getting this warning on the compilation (though there are many already being thrown).

[warn] core/src/test/scala/org/apache/spark/storage/StorageStatusListenerSuite.scala:174: reflective access of structural type member method advance should be enabled
[warn] by making the implicit value scala.language.reflectiveCalls visible.
[warn] This can be achieved by adding the import clause 'import scala.language.reflectiveCalls'
[warn] or by setting the compiler option -language:reflectiveCalls.
[warn] See the Scala docs for value scala.language.reflectiveCalls for a discussion
[warn] why the feature should be explicitly enabled.
[warn]     ticker.advance(5, TimeUnit.SECONDS)


That's why I imported that. I checked again, I am still getting this.